<?xml version="1.0" encoding="UTF-8"?>
<TestRun id="227b2372-b552-4662-99b8-37429de7ed0f" name="Huyoucai@HUYOUCAI-PC 2014-07-20 22:08:15" runUser="Huyoucai-PC\Huyoucai" xmlns="http://microsoft.com/schemas/VisualStudio/TeamTest/2010">
  <TestSettings name="Local" id="e4167082-9d56-471a-b488-96ac36bacfa7">
    <Description>These are default test settings for a local test run.</Description>
    <Deployment enabled="false" runDeploymentRoot="Huyoucai_HUYOUCAI-PC 2014-07-20 22_08_15">
      <DeploymentItem filename="G:\GitHub\Algorithm-Recursion-Dynamic\Algorithm-Recursion\bin\Debug\Algorithm-Recursion.dll" />
    </Deployment>
    <Execution>
      <TestTypeSpecific />
      <AgentRule name="Execution Agents">
      </AgentRule>
    </Execution>
  </TestSettings>
  <Times creation="2014-07-20T22:08:15.9550304-06:00" queuing="2014-07-20T22:08:17.3111080-06:00" start="2014-07-20T22:08:17.4321149-06:00" finish="2014-07-20T22:08:57.8264254-06:00" />
  <ResultSummary outcome="Error">
    <Counters total="1" executed="1" error="0" failed="0" timeout="0" aborted="1" inconclusive="0" passedButRunAborted="0" notRunnable="0" notExecuted="0" disconnected="0" warning="0" passed="0" completed="0" inProgress="0" pending="0" />
    <RunInfos>
      <RunInfo computerName="HUYOUCAI-PC" outcome="Error" timestamp="2014-07-20T22:08:57.8244252-06:00">
        <Text>Test host process exited unexpectedly.</Text>
      </RunInfo>
    </RunInfos>
  </ResultSummary>
  <TestDefinitions>
    <UnitTest name="Concatenation2Test" storage="g:\github\algorithm-recursion-dynamic\testproject\bin\debug\testproject.dll" id="64f1280f-e459-9ed0-160d-9af9d616f971">
      <Execution id="0e7a2b21-af0e-4782-9ce7-770d55a6c8a8" />
      <TestMethod codeBase="G:/GitHub/Algorithm-Recursion-Dynamic/TestProject/bin/Debug/TestProject.DLL" adapterTypeName="Microsoft.VisualStudio.TestTools.TestTypes.Unit.UnitTestAdapter, Microsoft.VisualStudio.QualityTools.Tips.UnitTest.Adapter, Version=10.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" className="TestProject.RecursionTest, TestProject, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null" name="Concatenation2Test" />
    </UnitTest>
  </TestDefinitions>
  <TestLists>
    <TestList name="Results Not in a List" id="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
    <TestList name="All Loaded Results" id="19431567-8539-422a-85d7-44ee4e166bda" />
  </TestLists>
  <TestEntries>
    <TestEntry testId="64f1280f-e459-9ed0-160d-9af9d616f971" executionId="0e7a2b21-af0e-4782-9ce7-770d55a6c8a8" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
  </TestEntries>
  <Results>
    <UnitTestResult executionId="0e7a2b21-af0e-4782-9ce7-770d55a6c8a8" testId="64f1280f-e459-9ed0-160d-9af9d616f971" testName="Concatenation2Test" computerName="HUYOUCAI-PC" duration="00:00:40.2753037" startTime="2014-07-20T22:08:17.5511217-06:00" endTime="2014-07-20T22:08:57.8264254-06:00" testType="13cdc9d9-ddb5-4fa4-a97d-d965ccfc6d4b" outcome="Aborted" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" relativeResultsDirectory="0e7a2b21-af0e-4782-9ce7-770d55a6c8a8">
    </UnitTestResult>
  </Results>
</TestRun>